pkg/cmd/codespace/common.go

```diff
@@ -284,3 +284,18 @@ func (c codespace) hasUnsavedChanges() bool {
 func (c codespace) running() bool {
 	return c.State == api.CodespaceStateAvailable
 }
+
+// addDeprecatedRepoShorthand adds a -r parameter (deprecated shorthand for --repo)
+// which instructs the user to use -R instead.
+func addDeprecatedRepoShorthand(cmd *cobra.Command, target *string) error {
```

Comment:
If there are any suggestions for how to make this cleaner, I would love to improve this 😄 I couldn't figure out a way to mark `-r` as deprecated and still have `-R` added

Reply:
This makes sense to me